test/kubernetes_unittest.cc

```diff
@@ -122,7 +122,7 @@ TEST_F(KubernetesTest, GetNodeMetadata) {
   EXPECT_EQ(time::rfc3339::FromString("2018-03-03T01:23:45.678901234Z"),
             m.metadata().created_at);
   EXPECT_EQ(Timestamp(), m.metadata().collected_at);
-  json::value big = json::object({
+  json::value node_metadata = json::object({
```

Comment:
Not directly related to this PR, but the static test data like `node_metadata` are repeated in the test file multiple times (in this case 4 times). WDYT about refactoring it into a constant? The only downside I could think of is that the `node_metadata` might not be in the same section as the test itself. But it's easier to maintain and we can focus on the test logic itself better (hopefully we don't have to scroll too much within one test).

Reply:
Refactoring it into a constant that's used verbatim should be fine, but the [style guide](https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables) does not favor non-POD static constants, and a `std::unique_ptr` would definitely be a non-POD. There are ways around it, but they might not be worth the bother.

Reply:
Sounds good

Reply:
FYI, I ended up factoring out some of the metadata into helper functions.